plugins/smqtk/smqtk_process_query.py

```diff
@@ -147,6 +151,41 @@ def add_config_traits(self):
         )
         self.declare_config_using_trait('query_return_size')
 
+    def get_svm_bytes(self):
+        svm_model = self.iqr_session.rel_index.get_model()
+        tmp_file_name = "tmp_svm.model"
+
+        svmutil.svm_save_model(tmp_file_name, svm_model)
+        with open(tmp_file_name, "rb") as f:
+            model_file = f.read()
+            b = bytearray(model_file)
+        os.remove(tmp_file_name)
+        return b
+
+    def get_model_from_bytes(self, bytes):
+        c_bytes = (ctypes.c_ubyte * len(bytes))(*bytes)
+        model = svmutil.svm_load_model_from_bytes(c_bytes)
+        return model
+
+    # (TODO (Mmanu)) Remove, intended for testing the code!
+    def test_model_from_byte(self):
+        # The original model
+        svm_model_1 = self.iqr_session.rel_index.get_model()
+        model_1_file, model_2_file = "tmp_svm_1.model", "tmp_svm_2.model"
+        svmutil.svm_save_model(model_1_file, svm_model_1)
+
+        # Get the bytes for the model first.
+        bytes = self.get_svm_bytes()
+        # Use the bytes to created a model
+        svm_model_2 = self.get_model_from_bytes(bytes)
+        # Save the model created using the bytes
+        svmutil.svm_save_model(model_2_file, svm_model_2)
+
+        # Check that the model created using the bytes is the same as the
+        # original model.
+        assert(filecmp.cmp(model_1_file, model_2_file) is True)
```

Comment:
nitpick: it would be more pythonic to write the assert this way:
`assert filecmp.cmp(model_1_file, model_2_file)`